package.json

```diff
@@ -32,7 +32,7 @@
   },
   "files": [
     "lib",
-    "src"
+    "docs"
```

Comment:
Also, why `docs` and not `documentation`? We usually try to avoid abbreviations, we even have a linting rule for it.

Reply:
solved in https://github.com/ridedott/eslint-plugin/pull/30/commits/11f36d49968150250492f1f27ebcb3dfe1764a4a